workers/unity/Assets/Playground/Scripts/Player/PlayerComponentTypes.cs

```diff
@@ -0,0 +1,8 @@
+using Unity.Entities;
```

Comment:
would rename that script to `ConnectButtonClicked`, if that's the only thing that's stored in this file